components/ArtPieces/ArtPieces.js

```diff
@@ -0,0 +1,24 @@
+import ArtPiecePreview from "../ArtPiecePreview/ArtPiecePreview";
+import styled from "styled-components";
+
+export function ArtPieces({ data }) {
+  return (
+    <StyledUl>
+      {data.map((artPiece) => {
+        return (
+          <li key={artPiece.slug}>
+            {" "}
+            <ArtPiecePreview artPiece={artPiece} />{" "}
+          </li>
+        );
+      })}
+    </StyledUl>
+  );
+}
+
+const StyledUl = styled.ul`
```

Comment:
You could avoid here the shortcut "ul" and better could choose a styled component name like "StyledList". This would be better for readability, or what do you think?

Reply:
Updated ✅